tests/core/conftest.py

```diff
@@ -63,6 +63,7 @@ def admin_mc():
         'password': 'admin',
         'responseType': 'json',
     }, verify=False)
+    protect_response(r)
```

Comment:
:+1:

Reply:
So interestingly, it seems that this first response always fails, since it's trying to change the password so that it can authenticate, but is not itself authenticated and (fortunately) you must authenticate to change the password. 

Do we just wanna remove that first request?